### DISCUSSION TITLE
Singleplayer supported?

### DISCUSSION THREAD

**Discussion by NaBrHCl** [iceiix/stevenarella#779] [Q&A]
Is it possible to enter a save in singleplayer mode, without setting up any server locally?

**Comment by ec3f6c47** [comment]
https://minecraft.wiki/w/Java_Edition_1.3.1#General_2
The minecraft client runs an internal server, which (I'm asusming) would be out of scope for this project.
Running PumpkinMC on localhost would work for 100% rust "singleplayer" if you don't mind not having the worlds menu.
